Utilities/DGraph/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@
 #
 ###########################################################################
 
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 2.8)
```

Comment:
This appears to be reverting recent commit https://github.com/commontk/CTK/commit/87b18d82a7898a5461d84c2146cc0fa60f1e1ba1. This diff here should be reverted to maintain that commit's changes.